src/Exception/ExceptionInterface.php

```diff
@@ -6,6 +6,8 @@
  * @license   https://github.com/laminas/laminas-json-server/blob/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace Laminas\Json\Server\Exception;
 
 interface ExceptionInterface
```

Comment:
We want to add `extends Throwable` here.

Reply:
Agreed. I will open an issue as reminder and provide changes separately.